src/v2/Apps/ViewingRoom/Routes/Statement/Components/ViewingRoomWorks.tsx

```diff
@@ -29,7 +30,7 @@ const ViewingRoomWorks: React.FC<ViewingRoomWorksProps> = ({
   return (
     <>
       <Flex>
-        {edges.slice(0, 2).map(({ node: artwork }, index) => {
+        {edges.map(({ node: artwork }, index) => {
```

Comment:
Figured instead of asking for all the edges and only using couple - better to request only as much as one needs.